tests/lax_numpy_test.py

```diff
@@ -302,7 +302,8 @@ def op_record(name, nargs, dtypes, shapes, rng_factory, diff_modes,
     op_record("true_divide", 2, all_dtypes, all_shapes, jtu.rand_nonzero,
               ["rev"], inexact=True),
     op_record("ediff1d", 3, [np.int32], all_shapes, jtu.rand_default, []),
-    op_record("unwrap", 1, float_dtypes, nonempty_nonscalar_array_shapes,
+    op_record("unwrap", 1, [t for t in float_dtypes if t != dtypes.bfloat16],
+              nonempty_nonscalar_array_shapes,
```

Comment:
Should we leave a TODO here, or are we fine no longer supporting `bfloat16` in `jnp.unwrap`?

Reply:
Added a TODO. `np.unwrap` itself actually works, it just cannot promote 2*pi to `bfloat16` since scalar -> bfloat16 float promotions work in fewer places in numpy 1.21.